--- web/static/js/api.js
+++ web/static/js/api.js
@@ -193,7 +193,12 @@
                 http: { port: 8080 }
             }
         },
-        route: { path: "" },
+        route: { 
+            path: "",
+            default: { enabled: true, host: "" },
+            internal: { enabled: false, host: "" },
+            external: { enabled: false, host: "" }
+        },
         resources: { limits: { cpu: "100m", memory: "128Mi" }, requests: { cpu: "10m", memory: "64Mi" } }
     };
 }
